packages/devtools_app/lib/src/screens/memory/memory_graph_model.dart

```diff
@@ -388,14 +393,14 @@ class HeapGraph {
 
 abstract class HeapGraphElement {
   /// Outbound references, i.e. this element points to elements in this list.
-  List<HeapGraphElement> _references;
+  List<HeapGraphElement?>? _references;
 
-  void Function() referencesFiller;
+  void Function()? referencesFiller;
 
-  List<HeapGraphElement> get references {
+  List<HeapGraphElement?>? get references {
```

Comment:
this can return a non-nullable list

Reply:
what if _references == null && referencesFiller == null

Reply:
ah good point.